tgui/packages/tgui-say/constants/index.tsx

```diff
@@ -1,6 +1,6 @@
 /** Radio channels */
-export const CHANNELS = ['Say', 'Radio', 'Me', 'OOC', 'LOOC'];
-export const NO_RADIO_CHANNELS = ['Me', 'OOC', 'LOOC'];
+export const CHANNELS = ['Say', 'Radio', 'Me', 'OOC', 'LOOC', 'Asay', 'Msay', 'Dsay'];
```

Comment:
Don't add these unless they are hidden when you don't have the applicable permissions. Ideally they should not show in tab/button clicking at all. They should only be accessible from the keybinds and if you press the bind it should lock the channel so it can't be changed until another key is pressed